libs/agno/tests/integration/tools/test_tool_generated_media.py

```diff
@@ -0,0 +1,219 @@
+import os
+
+import pytest
+
+from agno.agent.agent import Agent
+from agno.db.base import SessionType
+from agno.db.in_memory.in_memory_db import InMemoryDb
+from agno.media import AudioArtifact, ImageArtifact
+from agno.models.google import Gemini
+from agno.models.openai.chat import OpenAIChat
+from agno.tools.dalle import DalleTools
+
+
+@pytest.mark.integration
+@pytest.mark.skipif(not os.getenv("OPENAI_API_KEY"), reason="OPENAI_API_KEY not set")
+class TestAgentToolGeneratedMedia:
```

Comment:
We typically don't do test classes, but rather just use fixtures if you need something in a test. Rather use that pattern

Reply:
got it, i'll update